packages/lesswrong/components/search/PostsListEditorSearchHit.tsx

```diff
@@ -47,7 +47,7 @@ const PostsListEditorSearchHit = ({hit, classes}: {
         <PostsPreviewTooltipSingle postId={post._id} postsList/>
       </LWPopper>
       <div>
-        <PostsTitle post={post as unknown as PostsBase} isLink={false}/>
+        <PostsTitle post={post as unknown as PostsListBase} isLink={false}/>
```

Comment:
The `post` prop of `PostsTitle` previously had type `PostsBase`, which doesn't have a `tags` property. The expanded `PostsListBase` type does have access to tags, which I used to determine which icon to display. Every other use of the `PostsTitle` component passes in a `post` of type `PostsListBase` (or an extension thereof) except this one, and this case is just a workaround to avoid ts errors from `AlgoliaPost`. (The commit message for this line's last edit is "Awkwardly pretend AlgoliaPost isn't missing fields PostsTitle could use (but doesn't strictly need)".)

Reply:
Yep, seems fine. The fully-correct thing to do here would be to make `PostsTitle` take type `PostsListBase|AlgoliaPost`, and propagate that change through things that `PostsTitle` calls. Weird things like this, when they come up, are mainly artifacts of this code base's recent history as Javascript rather than Typescript.

Reply:
Alas, @jcmustin comment was wrong and the above did indeed directly cause a crash in the sequence editor. The code in `PostsTitle` did directly assume that the `tags` property was not undefined, and this component seems to now reliably crash. 

I applied the fix that Jim suggested in #4111